xml/soc-operations-gpu-passthrough.xml

```diff
@@ -128,18 +114,175 @@ drwxr-xr-x 3 root root 0 Feb 19 16:09 /sys/kernel/iommu_groups/20/devices/0000:0
      <note>
        <para>
          We suggest user to create new <filename>nova.conf</filename> file
-         specific to pci passthrough usage under
-         <literal>/etc/nova/nova.conf.d</literal> instead of modifying the
-         existing <filename>nova.conf</filename> to avoid any side effects
-         caused from the <literal>chef-client.service</literal>. For example,
-         <filename>100-nova.conf</filename> or
-         <filename>101-nova-placement.conf</filename>. Note that this
-         new <filename>nova.conf</filename> will not be recorded in any nova
+         specific to pci passthrough usage under<literal>/etc/nova/nova.conf.d</literal>
+         instead of modifying the existing <filename>nova.conf</filename> to avoid any side effects
+         caused from the <literal>chef-client.service</literal>. For example, <filename>100-nova.conf</filename> or
+         <filename>101-nova-placement.conf</filename>. Note that this new file, for example,
+         <filename>102-nova-pcipassthru.conf</filename> will not be recorded in any nova
          proposal and under manipulation by the <literal>chef-client.service</literal>.
          Ensure you have backups in case of any future incidents.
        </para>
      </note>
-   </listitem>
+     <para>Here's an <emphasis role="bold">example</emphasis> of the configuration of a single PCI device being
+       passthrough to the nova instance:</para>
+    <orderedlist>
+    <listitem>
+    <para>
+    <emphasis role="bold">Configure the Compute Host</emphasis>
+    </para>
+    <procedure>
+    <step>
+    <para>Login to the compute host and change directory to /etc/nova/nova.conf.d and list all the files
+    </para>
+<screen>&prompt.root;cd /etc/nova/nova.conf.d/
+ls -al
+</screen>
+    </step>
+    <step>
+    <para>Create a new nova configuration file under this folder and name the file appropriately in lexicographic
+    increasing order. For example: <filename>102-nova-pcipassthru.conf </filename>.
+    </para>
+    </step>
+    <step>
+    <para>
+    Add the following configuration entries to the file and the values for these entries are specific to your
+    compute node as obtained in Preparing the &compnode; section. The configuration entries specify the PCI device
```

Comment:
If you want to link to this specific section, you can use <xref linkend="SECTION-ID"/>

Reply:
Could you specify an example for this change?

Reply:
updated in the latest commit to include the link.